utils/encode exception in latest release

Create a issue in case someone found this issue too, I'm trying to find the root-cause. 
```sh
✘ [ERROR] [plugin checkForNodeBuiltins] Detected a Node builtin module import while Node compatibility is disabled.
Add node_compat = true to your wrangler.toml file to enable Node compatibility.

    node_modules/hono/dist/utils/encode.js:42:35:
      42 │         const { Buffer } = require('buffer');
         ╵                                    ~~~~~~~~

  This error came from the "onResolve" callback registered here:

    ../../../../.npm/_npx/dce38835dbc726fc/node_modules/wrangler/wrangler-dist/cli.js:109002:11:
      109002 │     build6.onResolve({
             ╵            ~~~~~~~~~

    at setup (/Users/minhuang/.npm/_npx/dce38835dbc726fc/node_modules/wrangler/wrangler-dist/cli.js:109002:12)
    at handlePlugins (/Users/minhuang/.npm/_npx/dce38835dbc726fc/node_modules/esbuild/lib/main.js:843:23)
    at Object.buildOrServe (/Users/minhuang/.npm/_npx/dce38835dbc726fc/node_modules/esbuild/lib/main.js:1137:7)
    at /Users/minhuang/.npm/_npx/dce38835dbc726fc/node_modules/esbuild/lib/main.js:2079:17
    at new Promise (<anonymous>)
    at Object.build (/Users/minhuang/.npm/_npx/dce38835dbc726fc/node_modules/esbuild/lib/main.js:2078:14)
    at Object.build (/Users/minhuang/.npm/_npx/dce38835dbc726fc/node_modules/esbuild/lib/main.js:1927:51)
    at bundleWorker (/Users/minhuang/.npm/_npx/dce38835dbc726fc/node_modules/wrangler/wrangler-dist/cli.js:109030:33)
    at build6 (/Users/minhuang/.npm/_npx/dce38835dbc726fc/node_modules/wrangler/wrangler-dist/cli.js:109124:75)


✘ [ERROR] Build failed with 1 error:

  node_modules/hono/dist/utils/encode.js:42:35: ERROR: [plugin: checkForNodeBuiltins] Detected a
  Node builtin module import while Node compatibility is disabled.
  Add node_compat = true to your wrangler.toml file to enable Node compatibility.
```

--- a/src/middleware/mustache/mustache.ts
+++ b/src/middleware/mustache/mustache.ts
@@ -1,3 +1,4 @@
+import Mustache from 'mustache'
 import type { Context } from '../../context'
 import type { Handler, Next } from '../../hono'
 import { bufferToString } from '../../utils/buffer'
@@ -23,13 +24,6 @@ export const mustache = (init: MustacheOptions = { root: '' }): Handler => {
   const { root } = init
 
   return async (c: Context, next: Next) => {
-    let Mustache: Mustache
-    try {
-      Mustache = require('mustache')
-    } catch {
-      throw new Error('If you want to use Mustache Middleware, install "mustache" package first.')
-    }
-
     c.render = async (filename, params = {}, options?) => {
       const path = getKVFilePath({
         filename: `${filename}${EXTENSION}`,

--- a/src/utils/crypto.ts
+++ b/src/utils/crypto.ts
@@ -30,13 +30,4 @@ export const createHash = async (data: Data, algorithm: Algorithm): Promise<stri
       .join('')
     return hash
   }
-
-  try {
-    const crypto = require('crypto')
-    const hash = crypto.createHash(algorithm.alias).update(data).digest('hex')
-    return hash
-  } catch (e) {
-    console.error(`If you want to create hash ${algorithm.name}, polyfill "crypto" module.`)
-    throw e
-  }
 }

--- a/src/utils/encode.ts
+++ b/src/utils/encode.ts
@@ -9,7 +9,6 @@ export const encodeBase64 = (str: string): string => {
   } catch {}
 
   try {
-    const { Buffer } = require('buffer')
     return Buffer.from(str).toString('base64')
   } catch (e) {
     console.error('If you want to do "encodeBase64", polyfill "buffer" module.')
@@ -29,7 +28,6 @@ export const decodeBase64 = (str: string): string => {
   } catch {}
 
   try {
-    const { Buffer } = require('buffer')
     return Buffer.from(str, 'base64').toString()
   } catch (e) {
     console.error('If you want to do "decodeBase64", polyfill "buffer" module.')
@@ -67,7 +65,6 @@ export const arrayBufferToBase64 = async (buf: ArrayBuffer): Promise<string> => 
   }
 
   try {
-    const { Buffer } = await import('buffer')
     return Buffer.from(String.fromCharCode(...new Uint8Array(buf))).toString('base64')
   } catch (e) {}
 }
@@ -75,4 +72,3 @@ export const arrayBufferToBase64 = async (buf: ArrayBuffer): Promise<string> => 
 export const arrayBufferToBase64URL = async (buf: ArrayBuffer) => {
   return (await arrayBufferToBase64(buf)).replace(/=/g, '').replace(/\+/g, '-').replace(/\//g, '_')
 }
-

